### DISCUSSION TITLE
机械师t58-能帮忙制作EFI吗

### DISCUSSION THREAD

**Discussion by letVV** [daliansky/Hackintosh#375] [Q&A]
![硬件参数](https://user-images.githubusercontent.com/43233108/155470903-78130b4b-67ab-4ced-b742-a8e4ca985547.png)

**Comment by swanduron** [comment]
显卡限定了最高只能安装10.13版本。10.13以后，nVidia不再提供基于macOS的驱动程序。

**Reply by letVV** [reply]
就是制作不了EFI吗

**Reply by letVV** [reply]
我的也有集显的

**Reply by letVV** [reply]
电脑型号               MACHENIKE T58 笔记本电脑
  操作系统               Windows 10 专业版 64位（Version 20H2 / DirectX 12）

  处理器                 英特尔 Core i7-8750H @ 2.20GHz 六核
  主板                   MACHENIKE N855EX-HM（HM370 芯片组）
  显卡                   NVIDIA GeForce GTX 1050 Ti ( 4 GB / 蓝天(CLEVO) )
  内存                   8 GB ( 三星 DDR4 2400MHz )
  主硬盘                 英特尔 SSDSCKHB340G4M 118000041 ( 340 GB / 固态硬盘 )
  显示器                 戴尔 DEL4185 DELL P2719H ( 27 英寸  )
  声卡                   瑞昱 ALC269 @ 英特尔 High Definition Audio 控制器
  网卡                   英特尔 Wireless-AC 9462

--------[ 主板 ]----------------------------------------------------------------------------------

  主板型号               MACHENIKE N855EX-HM
  芯片组                 HM370 芯片组
  序列号                 JJ00G100000JMJAG0027
  主板版本               Not Applicable
  BIOS                   American Megatrends Inc. 7.007  /  BIOS程序发布日期: 08/03/2018
  BIOS的大小             8192 KB

  板载设备               HD-Audio / 音频设备 (启用)
  板载设备               Giga Lan / 网卡 (启用)

--------[ 处理器 ]----------------------------------------------------------------------------------

  处理器                 英特尔 Core i7-8750H @ 2.20GHz 六核
  速度                   2.20 GHz
  处理器数量             核心数：6 / 线程数：12
  核心代号               Coffee Lake
  生产工艺               14 nm
  插槽/插座              FCBGA1440
  一级数据缓存           6 x 32 KB, 8-Way, 64 byte lines
  一级代码缓存           6 x 32 KB, 8-Way, 64 byte lines
  二级缓存               6 x 256 KB, 4-Way, 64 byte lines
  三级缓存               9 MB, 12-Way, 64 byte lines
  特征                   MMX, SSE, SSE2, SSE3, SSSE3, SSE4.1, SSE4.2, HTT, EM64T, EIST, Turbo Boost

--------[ 硬盘 ]----------------------------------------------------------------------------------

  产品                   英特尔  SSDSCKHB340G4M           118000041 (固态硬盘)
  大小                   340 GB
  固件                   G201EM05
  接口                   SATA 2.6
  数据传输率             300 MB/秒
  特性                   S.M.A.R.T,  48-bit LBA,  NCQ
  硬盘已使用             共 2537 次，累计 7440 小时

  产品                   希捷 ST1000LM048-2E7172
  大小                   1 TB
  固件                   SDM1
  缓存                   128 MB
  接口                   SATA III
  数据传输率             600.00 MB/秒
  特性                   S.M.A.R.T,  APM,  48-bit LBA,  NCQ
  硬盘已使用             共 2538 次，累计 7446 小时
  转速                   5400 转/分

--------[ 内存 ]----------------------------------------------------------------------------------

  ChannelA-DIMM0         三星 DDR4 2400MHz 8GB
  制造日期               2018 年 32 周
  型号                   M471A1K43CB1-CRC
  序列号                 404386A1
  厂商                   SAMSUNG
  模块位宽               64 Bits
  模块电压               SSTL 1.2V

--------[ 显卡 ]----------------------------------------------------------------------------------

  主显卡                 NVIDIA GeForce GTX 1050 Ti
  显存                   4 GB
  显卡制造商             蓝天(CLEVO)
  芯片制造商             Nvidia
  驱动版本               27.21.14.5720
  驱动日期               20201102

  显卡                   英特尔 UHD Graphics 630
  显存                   128 MB
  显卡制造商             蓝天(CLEVO)
  芯片制造商             Intel
  驱动版本               27.20.100.8681
  驱动日期               20200905

--------[ 显示器 ]----------------------------------------------------------------------------------

  产品                   友达 AUO61ED
  厂商                   友达
  固件程序日期           2016 年
  屏幕尺寸               15.5 英寸 (344 毫米 x 193 毫米)
  显示比例               宽屏 16 : 9
  分辨率                 1920 x 1080 (缩放比例:125%) 32 位真彩色
  Gamma                  2.20

  产品                   戴尔 DEL4185 DELL P2719H
  厂商                   戴尔
  固件程序日期           2020 年 49 周
  屏幕尺寸               27 英寸 (598 毫米 x 336 毫米)
  显示比例               宽屏 16 : 9
  分辨率                 1920 x 1080 32 位真彩色
  Gamma                  2.20
  电源管理               Standby, Suspend, Active-Off

--------[ 其他设备 ]----------------------------------------------------------------------------------

  网卡                   Intel(R) Wireless-AC 9462

  网卡                   Realtek PCIe GbE Family Controller

  声卡                   瑞昱 ALC269 @ 英特尔 High Definition Audio 控制器

  键盘                   PS/2 标准键盘
  鼠标                   HID-compliant 鼠标
  鼠标                   Synaptics SMBus 触摸板
  鼠标                   HID-compliant 鼠标
